### PR TITLE
Update: Internet_Relay_Chat\en.md

### PR DESCRIPTION
**Pull request by Nath789**
### Description
Another rewritten article with some cleanup

### Status
Finished

### Changelog
* Fixed table of IRC clients showing how to disable join/parts.
* Some housekeeping (removing unused `<center>` html, etc.)

### CODE REVIEW

File: wiki/Internet_Relay_Chat/en.md [hunk 1]
```diff
@@ -1,100 +1,84 @@
 What is Internet Relay Chat?
 ============================
 
-The [Internet Relay Chat](http://en.wikipedia.org/wiki/Internet_Relay_Chat), or also known as IRC is a well established standardized protocol for chatting with numerous clients available to connect with.
+The [Internet Relay Chat](http://en.wikipedia.org/wiki/Internet_Relay_Chat), also known as IRC is a well established standardized protocol for chatting with numerous clients available to connect with.
```
Comment (MegaApplePi) [inline wiki/Internet_Relay_Chat/en.md]: I think there is a missing comma:
```
The [Internet Relay Chat](http://en.wikipedia.org/wiki/Internet_Relay_Chat), also known as IRC is a well established standardized protocol for chatting with numerous clients available to connect with.
                                                                                              ^
                                                                                              here
```

File: wiki/Internet_Relay_Chat/en.md [hunk 2]
```diff
@@ -1,100 +1,84 @@
...
 Disabling Join/Part Messages
 ============================
-
-Every time someone joins a channel or leaves it a message like this is shown:
-
-`someuser has joined #somechannel`
-`someuser has quit #somechannel`
-
-While in low traffic channels this is usually isn't disruptive, people are joining and leaving \#osu constantly making it potentially hard to keep track of the chat.
+Knowing players who joins or parts is a good thing but in very crowded channels such as #osu, you will be spammed very much and you can't keep track of the conversations. Therefore, these must be disabled.
```
Comment (ElectrifyPro) [inline wiki/Internet_Relay_Chat/en.md]: "...is a good thing, however in very crowded channels such as #osu, you will constantly receive join/part messages and will not be able to keep track of conversations. Therefore..."

sounds more professional

File: wiki/Internet_Relay_Chat/en.md [hunk 3]
```diff
@@ -1,100 +1,84 @@
...
+| [ircII](http://www.eterna.com.au/ircii/)  | Type `/ignore * crap` |
+| [Irssi](http://www.irssi.org)             | Type `/ignore -channels #somechannel * JOINS PARTS QUITS` |
+| [Weechat](http://www.weechat.org)         | Type `/filter add irc_smart_weechat irc.username.#channel irc_smart_filter *` <br> **Note:** Replace **username** to your osu! username.
+| [KVIrc](http://www.kvirc.net)             | Visit [this thread](http://www.kvirc.ru/forum/?topic=609.0) in the official KVIrc forums. |
+| [mIRC](http://www.mirc.com/)              | Go to mIRC Options (Tools - Options / Alt + O), under IRC tree, click "Events..." button and change Joins and Parts to "Hide" |
+| [Quassel IRC](http://www.quassel-irc.org) | Right click on the chatting window, then choose Hide Events » Join/Part/Quit. |
+| [XChat](http://www.xchat.org)             | Type `/set irc_conf_mode 1` (or [2](http://xchat.org/faq/#q211) to disable the messages throughout the channels). |
 
 If your client isn't listed here refer to its documentation, most clients have a way to do this.
```
Comment (ElectrifyPro) [inline wiki/Internet_Relay_Chat/en.md]: "If your client isn't listed here, refer to its documentation. Most clients will have an option to disable these messages."

File: wiki/Internet_Relay_Chat/en.md [hunk 4]
```diff
@@ -1,100 +1,84 @@
...
 
+
 Frequently Asked Questions (FAQ)
 ================================
-
-I am having an error "Bad Authentication Token". What should I do?
+I am having an error "Bad Authentication Token".
```
Comment (ElectrifyPro) [inline wiki/Internet_Relay_Chat/en.md]: I'm receiving the error "Bad Authentication Token".

File: wiki/Internet_Relay_Chat/en.md [hunk 5]
```diff
@@ -1,100 +1,84 @@
...
 ------------------------------------------------------------------
-
 1. Make sure you are using the password at the [IRC Authentication](https://osu.ppy.sh/p/irc) page.
+2. If your username had a space, replace it to underscores (e.g. **This Username** to **This_Username**)
```
Comment (ElectrifyPro) [inline wiki/Internet_Relay_Chat/en.md]: "If your username has spaces, replace them with underscores (e.g. **This Username..."

File: wiki/Internet_Relay_Chat/en.md [hunk 6]
```diff
@@ -1,100 +1,84 @@
...
-No. You can only use the username that you're using ingame.
 
 What is this voice status that I have? I also see some people having it.
 ------------------------------------------------------------------------
+Users with *voice status* are also connected using an IRC client, except for chat moderators who always have *operator (+o)* status regardless of what client they use.
 
-Users with voice (prefix +) status are connected using an IRC client (except the chat moderators who have *operator (prefix @)* status).
-
-Users who have no mode are basically who are connected using the game client.
+Users who doesn't have any mode are basically connected using the game client.
```
Comment (ElectrifyPro) [inline wiki/Internet_Relay_Chat/en.md]: "Users who don't have any mode are connected using the game client."

File: wiki/Internet_Relay_Chat/en.md (hunk 3 above)
Comment (kj415j45) [reply wiki/Internet_Relay_Chat/en.md]: ASG: `isn't` -> `is not`

File: wiki/Internet_Relay_Chat/en.md (hunk 6 above)
Comment (kj415j45) [reply wiki/Internet_Relay_Chat/en.md]: ASG: doesn't -> does not

**Comment by Nath789** [comment]
Done. Kindly recheck.

File: wiki/Internet_Relay_Chat/en.md [hunk 7]
```diff
@@ -1,100 +1,84 @@
...
-When you go to the URL you are presented with you'll see a screen with a big fancy "Authorise IRC connection" button. Simply clicking this will allow you into the chat and auto-join you into [\#osu](irc://cho.ppy.sh/osu).
-
-If you wish to not have to click the big fancy button every time you connect you can put the password given into either your server's password field in a profile or use it while connecting.
-
-`To permanently authorise a client, please change your IRC password (server password) to: XXXXXXX`
-
-Remember **NOT TO GIVE YOUR IRC PASSWORD** to anyone else.
 
 Basic IRC Commands
 ==================
```
Comment (MegaApplePi) [inline wiki/Internet_Relay_Chat/en.md]: Use heading level 2, there should only be one heading level 1, that being the title.
```
Basic IRC Commands
------------------
```

File: wiki/Internet_Relay_Chat/en.md [hunk 8]
```diff
@@ -1,100 +1,84 @@
...
 
-| Description                 | Command             |
-|-----------------------------|---------------------|
-| Join channel (I.E. \#lobby) | /join \#somechannel |
-| Leave channel               | /part               |
-| Ignore nick                 | /ignore somenick    |
-| Make actions                | /me does something  |
 
 Disabling Join/Part Messages
 ============================
```
Comment (MegaApplePi) [inline wiki/Internet_Relay_Chat/en.md]: ditto
```
Disabling Join/Part Messages
----------------------------
```

File: wiki/Internet_Relay_Chat/en.md [hunk 9]
```diff
@@ -1,100 +1,84 @@
...
+[17:46] * AuReL (cho@ppy.sh) has joined #lobby
+[17:46] * osukd (cho@ppy.sh) has joined #lobby
+[17:46] * BreadTooGood (cho@ppy.sh) has joined #lobby
+[17:46] * keanyew18 (cho@ppy.sh) has joined #lobby
+[17:46] * JaKox (cho@ppy.sh) has joined #lobby
+[17:46] * Kerantor (cho@ppy.sh) has joined #lobby
+```
 
 Disabling Join/Part Messages In Common Clients
 ----------------------------------------------
```
Comment (MegaApplePi) [inline wiki/Internet_Relay_Chat/en.md]: If you do change the heading levels for `## Disabling Join/Part Messages`, then I assume that this one should also go down a level
```
### Disabling Join/Part Messages In Common Clients
```

File: wiki/Internet_Relay_Chat/en.md [hunk 10]
```diff
@@ -1,100 +1,84 @@
...
-| [Quassel IRC](http://www.quassel-irc.org) | Right click on the chatting window, then choose Hide Events » Join/Part/Quit.                                                                                                                                                                                            |
-| [XChat](http://www.xchat.org)             | Right-click on the tab you want to change. In the submenu of the channelname, there's a toggle-item "Show join/part messages", simply turn this off. Or you type /set irc\_conf\_mode 1 [2](http://xchat.org/faq/#q211) to disable the messages throughout the channels. |
+If your client is not listed here refer to its documentation, most clients have a way to do this.
 
-If your client isn't listed here refer to its documentation, most clients have a way to do this.
 
 Frequently Asked Questions (FAQ)
 ================================
```
Comment (MegaApplePi) [inline wiki/Internet_Relay_Chat/en.md]: Use heading level 2, there should only be one heading level 1, that being the title.
```
Frequently Asked Questions (FAQ)
--------------------------------
```

File: wiki/Internet_Relay_Chat/en.md [hunk 11]
```diff
@@ -1,100 +1,84 @@
...
-
-I am having an error "Bad Authentication Token". What should I do?
+I'm receiving the error "Bad Authentication Token".
 ------------------------------------------------------------------
```
Comment (MegaApplePi) [inline wiki/Internet_Relay_Chat/en.md]: If you do change the heading levels for `## Disabling Join/Part Messages`, then this one should also go down a level
```
### I'm receiving the error "Bad Authentication Token".
```

File: wiki/Internet_Relay_Chat/en.md [hunk 12]
```diff
@@ -1,100 +1,84 @@
...
-
 1. Make sure you are using the password at the [IRC Authentication](https://osu.ppy.sh/p/irc) page.
+2. If your username has spaces, replace it to underscores (e.g. **This Username** to **This_Username**)
 
-2. If your username had a space, replace it to underscores (e.g. **This Username** to **This\_Username**)
 
 Can I use another username?
 ---------------------------
```
Comment (MegaApplePi) [inline wiki/Internet_Relay_Chat/en.md]: ditto
```
### Can I use another username?
```

File: wiki/Internet_Relay_Chat/en.md [hunk 13]
```diff
@@ -1,100 +1,84 @@
...
+No. You can only use your osu! username.
 
-No. You can only use the username that you're using ingame.
 
 What is this voice status that I have? I also see some people having it.
```
Comment (MegaApplePi) [inline wiki/Internet_Relay_Chat/en.md]: ditto
```
### What is this voice status that I have? I also see some people having it.
```

File: wiki/Internet_Relay_Chat/en.md [hunk 14]
```diff
@@ -1,100 +1,81 @@
...
+[17:46] * Kastun (cho@ppy.sh) has joined #lobby
+[17:46] * AuReL (cho@ppy.sh) has joined #lobby
+[17:46] * osukd (cho@ppy.sh) has joined #lobby
+[17:46] * BreadTooGood (cho@ppy.sh) has joined #lobby
+[17:46] * keanyew18 (cho@ppy.sh) has joined #lobby
+[17:46] * JaKox (cho@ppy.sh) has joined #lobby
+[17:46] * Kerantor (cho@ppy.sh) has joined #lobby
+```
+
+## Disabling Join/Part Messages In Common Clients
```
Comment (MegaApplePi) [inline wiki/Internet_Relay_Chat/en.md]: `### Disabling Join/Part Messages In Common Clients` ❓

**Comment by Nath789** [comment]
Done. Anything else?

**Comment by sr229** [comment]
@TPGPL or @Repflez, kindly do final checking then merge, ty

File: wiki/Internet_Relay_Chat/en.md [hunk 15]
```diff
@@ -1,100 +1,81 @@
...
-| Leave channel               | /part               |
-| Ignore nick                 | /ignore somenick    |
-| Make actions                | /me does something  |
 
 Disabling Join/Part Messages
-============================
-
-Every time someone joins a channel or leaves it a message like this is shown:
+----------------------------
+Knowing players who joins or parts is a good thing, however in very crowded such as #osu, you will constantly receive join/part messages and will not be able to keep track of conversations. Therefore, these must be disabled.
```
Comment (RaikireHiuduo) [inline wiki/Internet_Relay_Chat/en.md]: #osu -> `#osu` (Add grave marks around it).

Comment (Nath789) [reply wiki/Internet_Relay_Chat/en.md]: Done.

File: wiki/Internet_Relay_Chat/en.md [hunk 16]
```diff
@@ -1,100 +1,81 @@
...
 Frequently Asked Questions (FAQ)
-================================
-
-I am having an error "Bad Authentication Token". What should I do?
-------------------------------------------------------------------
+--------------------------------
 
+### I'm receiving the error "Bad Authentication Token".
 1. Make sure you are using the password at the [IRC Authentication](https://osu.ppy.sh/p/irc) page.
+2. If your username has spaces, replace it to underscores (e.g. **This Username** to **This_Username**)
```
Comment (kj415j45) [inline wiki/Internet_Relay_Chat/en.md]: has space characters

replace them to

File: wiki/Internet_Relay_Chat/en.md [hunk 17]
```diff
@@ -1,100 +1,81 @@
...
 
 Basic IRC Commands
-==================
+------------------
+| Command              | Description                           |
+| -------------------- | ------------------------------------- |
+| `/join <#channel>`   | Joins a channel                       |
+| `/part <#channel>`   | Leaves a channel                      |
+| `/me <action>`       | Sends an action message               |
+| `/ignore <username>` | Ignores a user (hides their messages) |
```
Comment (kj415j45) [inline wiki/Internet_Relay_Chat/en.md]: join

leave

send

ignore

File: wiki/Internet_Relay_Chat/en.md [hunk 18]
```diff
@@ -1,100 +1,81 @@
...
 
-<center>
-</center>
-osu! Chat
-=========
 
-osu! uses the IRC protocol for its [in-game chat](Chat_Console "wikilink"). You can connect with your own client and chat with your osu! buddies even when you're not running osu!. Note that osu! Bancho uses a custom implementation of the IRC protocol, and is in no way a full-feature implementation; therefore, do not expect all features of your IRC client to function properly.
+osu!Bancho
+---------
+osu!Bancho uses the IRC protocol for its in-game chat. You can connect with your own client and chat with other people without using the game client. Take note that this IRC protocol is modified therefore do not expect your client's features to work properly.
```
Comment (peppy) [inline wiki/Internet_Relay_Chat/en.md]: no it doesn't. it offers a gateway for IRC access

File: wiki/Internet_Relay_Chat/en.md [hunk 19]
```diff
@@ -1,100 +1,81 @@
...
-Authenticating To Bancho
-------------------------
-
-When you first connect you'll get a message like this.
-
-`* Welcome to osu!bancho.`
-`* -`
-`* - You are required to authenticate before accessing this service.`
-`* - Please click the following link to complete this process:`
+- **Server:** `cho.ppy.sh` or `irc.ppy.sh`
```
Comment (peppy) [inline wiki/Internet_Relay_Chat/en.md]: just irc here please

File: wiki/Internet_Relay_Chat/en.md [hunk 20]
```diff
@@ -1,100 +1,81 @@
...
-| Leave channel               | /part               |
-| Ignore nick                 | /ignore somenick    |
-| Make actions                | /me does something  |
 
 Disabling Join/Part Messages
-============================
-
-Every time someone joins a channel or leaves it a message like this is shown:
+----------------------------
+Knowing players who joins or parts is a good thing, however in very crowded such as `#osu`, you will constantly receive join/part messages and will not be able to keep track of conversations. Therefore, these must be disabled.
```
Comment (peppy) [inline wiki/Internet_Relay_Chat/en.md]: "Therefore, it's usually preferred that these messages are hidden."

File: wiki/Internet_Relay_Chat/en.md [hunk 21]
```diff
@@ -1,100 +1,81 @@
...
 
-What is this voice status that I have? I also see some people having it.
-------------------------------------------------------------------------
 
-Users with voice (prefix +) status are connected using an IRC client (except the chat moderators who have *operator (prefix @)* status).
+### What is this voice status that I have? I also see some people having it.
+Users with *voice status* are also connected using an IRC client, except for chat moderators who always have *operator (+o)* status regardless of what client they use.
 
-Users who have no mode are basically who are connected using the game client.
+Users who does not have any status are connected using the game client.
```
Comment (peppy) [inline wiki/Internet_Relay_Chat/en.md]: "Users who do not"

**Pull request merged**